index.bikeshed.bs

```diff
@@ -343,7 +354,47 @@ implementation detail. It is not an API that implementors
 provide to web applications. It is used in this document
 only to simplify the algorithm description.
 
-## Response verification algorithms ## {#verification-algorithms}
+## Request verification algorithms ## {#request-verification-algorithms}
```

Comment:
It is still there https://github.com/shekyan/webappsec-subresource-integrity/blob/require-sri-for/index.bikeshed.bs#L397